packages/win_toast/windows/notification_manager.cc

```diff
@@ -3,33 +3,64 @@
 //
 
 #include <Windows.h>
+#include <appmodel.h>
+#include <malloc.h>
+#include <stdio.h>
 #include <hstring.h>
 #include <minappmodel.h>
 
 #include "notification_manager.h"
 #include "dll_importer.h"
 
-namespace {
+namespace
+{
 
-bool _checkedHasIdentity = false;
-bool _hasIdentity = false;
+  bool _checkedHasIdentity = false;
+  bool _hasIdentity = false;
 
-bool hasIdentity() {
-  // https://stackoverflow.com/questions/39609643/determine-if-c-application-is-running-as-a-uwp-app-in-desktop-bridge-project
-  UINT32 length;
-  wchar_t packageFamilyName[PACKAGE_FAMILY_NAME_MAX_LENGTH + 1];
-  LONG result = DllImporter::GetPackageFamilyName(GetCurrentProcess(), &length, packageFamilyName);
-  return result == ERROR_SUCCESS;
-}
+  // https://learn.microsoft.com/en-us/windows/msix/detect-package-identity
+  bool hasIdentity()
+  {
+    UINT32 length = 0;
+    LONG rc = GetCurrentPackageFullName(&length, NULL);
```

Comment:
Sorry for later, but seesm the `GetCurrentPackageFullName` wouldn't working on Win7.